SPECS/cloud-init/cloud-init.spec

```diff
@@ -163,8 +164,8 @@ rm -rf %{buildroot}
 %config(noreplace) %{_sysconfdir}/cloud/cloud.cfg.d/10-azure-kvp.cfg
 
 %changelog
-* Thu Jun 15 2023 Minghe Ren <mingheren@microsoft.com> - 21.4-3
-- Add patch for CVE-2023-1786
+* Tue Mar 22 2023 Minghe Ren <mingheren@microsoft.com> - 21.4-3
```

Comment:
Should this update be a release number increment to `-4`?

Reply:
I am thinking if I could do two patch change together as the previous PR for CVE-2023-1786 just merged but increment it makes it clearer.